gp_align/cli.py

```diff
@@ -92,17 +96,20 @@ def analyze(pattern, scanner, plate_type, orientation, out, time_unit,
         LOGGER.critical("No files match the given glob pattern.")
         return 1
     data = analyze_run(filenames, scanner, plate_type, orientation=orientation,
-                       unit=time_unit, num_proc=processes)
+                       plates=trays, unit=time_unit, num_proc=processes)
 
     for name, df in iteritems(data):
         df.to_csv(out + "_" + name + ".G.tsv", sep="\t")
 
 
 @cli.command()
 @click.help_option("--help", "-h")
+@click.option("--out", "-o", default=None, type=str,
+              help="The desired output filename. Only permissible when "
+                   "processing one file otherwise it is ignored.")
 @click.argument("parameters", nargs=3)
 @click.argument("pattern", type=str)
-def convert(pattern, parameters):
+def convert(pattern, parameters, out):
```

Comment:
Sorry if I'm just blind, but is `out` actually being used when writing the file?

Reply:
Weird, could have sworn that I had it working but it doesn't look like it right now.